package.json

```diff
@@ -1,73 +1,85 @@
-    "build-demo-app": "npm run build; cd packages/browser/examples/demoClientApp/; npm ci; npm run build; mkdir -p ../../../../dist/website/demo/; cp -r dist/. ../../../../dist/website/demo/"
+    "test:e2e:browser": "cd e2e/browser && npm test"
   },
   "dependencies": {
-    "lerna": "^4.0.0"
+    "@inrupt/oidc-client-ext": "file:packages/oidc",
+    "@inrupt/solid-client-authn-browser": "file:packages/browser",
+    "@inrupt/solid-client-authn-core": "file:packages/core",
+    "@inrupt/solid-client-authn-node": "file:packages/node",
+    "e2e-browser": "file:e2e/browser"
   },
   "devDependencies": {
-    "@inrupt/eslint-config-base": "0.1.2",
-    "@inrupt/eslint-config-lib": "0.4.0",
+    "@inrupt/eslint-config-base": "0.4.0",
+    "@inrupt/eslint-config-lib": "0.4.1",
+    "@inrupt/solid-client": "^1.20.1",
+    "@inrupt/solid-client-authn-node": "file:../../packages/node",
+    "@inrupt/vocab-common-rdf": "^1.0.3",
+    "@inrupt/vocab-inrupt-common": "^0.7.2",
+    "@playwright/test": "^1.18.0",
+    "@rollup/plugin-commonjs": "^21.1.0",
+    "@rollup/plugin-node-resolve": "^13.2.1",
     "@types/dotenv-flow": "^3.1.0",
     "@types/jest": "^27.0.3",
-    "@typescript-eslint/eslint-plugin": "^4.14.2",
-    "@typescript-eslint/parser": "^4.14.2",
+    "@types/n3": "^1.4.4",
+    "@typescript-eslint/eslint-plugin": "^5.14.0",
+    "@typescript-eslint/parser": "^5.10.1",
+    "cross-fetch": "^3.0.6",
     "dotenv-flow": "^3.2.0",
     "eslint": "^7.19.0",
     "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-config-prettier": "^7.2.0",
+    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^26.0.0",
     "eslint-plugin-license-header": "^0.4.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "husky": "^7.0.0",
     "jest": "^27.4.3",
+    "jest-location-mock": "^1.0.9",
+    "jest-mock-console": "^1.2.3",
+    "jose": "^4.3.7",
+    "lerna": "^4.0.0",
     "lerna-audit": "^1.3.3",
     "license-checker": "^25.0.1",
-    "lint-staged": "^12.1.2",
+    "n3": "^1.6.4",
     "prettier": "2.6.2",
     "rimraf": "^3.0.2",
     "rollup": "^2.38.4",
+    "rollup-plugin-polyfill-node": "^0.9.0",
+    "rollup-plugin-sourcemaps": "^0.6.3",
     "rollup-plugin-typescript2": "^0.31.1",
     "ts-jest": "^27.0.7",
     "ts-loader": "^9.1.1",
+    "ts-node": "^10.4.0",
     "tsc-watch": "^5.0.2",
     "typedoc": "^0.22.4",
+    "typedoc-plugin-markdown": "^3.10.2",
     "typescript": "^4.0.5",
-    "webpack": "^5.19.0",
-    "webpack-cli": "^4.1.0",
-    "webpack-merge": "^5.2.0"
-  },
-  "lint-staged": {
-    "*.{ts,js,css,json}": [
-      "prettier --write"
-    ]
+    "webpack": "^5.11.0",
+    "webpack-cli": "^4.2.0",
+    "webpack-merge": "^5.7.2"
```

Comment:
Webpack is only used for the example apps now, but we can and probably should migrate to something else there (swc, next, rollup, etc)